File: NotificationHub/Models/NotificationDto.cs
namespace NotificationHub.Models;

public class NotificationDto
{
    public int Id { get; set; }
    public int? ClaveId { get; set; }
    public int? SolicitudId { get; set; }
    public string? NombreProceso { get; set; }
    public string? Titulo { get; set; }
    public string? Mensaje { get; set; }
    public string? NotificationTipo { get; set; }
    public bool IsAcknowledged { get; set; }
    public string? RelativeNotifiedDateAndTime { get; set; }
    public DateTime CreatedOnUtc { get; set; }
    public int? Estado { get; set; }
    public string? Usuario { get; set; }
    public int? Prioridad { get; set; }
}

File: vue-client/src/types/NotificationDto.ts
export interface NotificationDto {
  id: number;
  claveId: number | null;
  solicitudId: number | null;
  nombreProceso: string | null;
  titulo: string | null;
  mensaje: string | null;
  notificationTipo: string | null;
  isAcknowledged: boolean;
  relativeNotifiedDateAndTime: string | null;
  createdOnUtc: Date;
  estado: number | null;
  usuario: string | null;
  prioridad: number | null;
}
